tee-worker/litentry/core/assertion-build/src/dynamic/README.md

```diff
@@ -0,0 +1,53 @@
+## Description
+
+Dynamic VC assertion contract written by solidity, using [Hardhat](https://hardhat.org) for compile and test.
+
+## Environment setup
+
+-   Install [nvm](https://github.com/nvm-sh/nvm)
+-   Inside the repository, run `nvm use` to set the correct Node version.
+    -   If the version is not installed, run `nvm install`.
+
+## Installation
+
+```shell
+nvm use
+corepack enable pnpm
+pnpm install
```

Comment:
I don't know why but I always must use `corepack pnpm install` 🤔

Reply:
See if below works for you
```
~/repo/litentry-parachain/tee-worker/litentry/core/assertion-build/src/dynamic$ nvm use
Found '/home/will/repo/litentry-parachain/.nvmrc' with version <lts/iron>
Now using node v20.16.0 (npm v10.8.1)
~/repo/litentry-parachain/tee-worker/litentry/core/assertion-build/src/dynamic$ pnpm install
Command 'pnpm' not found, did you mean:
  command 'npm' from deb npm (8.5.1~ds-1)
Try: sudo apt install <deb name>
~/repo/litentry-parachain/tee-worker/litentry/core/assertion-build/src/dynamic$ corepack enable pnpm
~/repo/litentry-parachain/tee-worker/litentry/core/assertion-build/src/dynamic$ pnpm install
Lockfile is up to date, resolution step is skipped
Already up to date
Done in 612ms
~/repo/litentry-parachain/tee-worker/litentry/core/assertion-build/src/dynamic$ pnpm -v
8.7.6
```